### ISSUE TITLE
Image upload fails after upgrade to 0.11.2

### ISSUE THREAD

**Issue by kkretsch** [pixelfed/pixelfed#3429]
I'm pretty sure it worked some while ago using 0.11.1, but after upgrading I cannot upload anymore.
`
2a02:xxx - - [08/May/2022:21:25:35 +0200] "POST /api/compose/v0/media/upload HTTP/2.0" 500 60 "https://p.xxx/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:100.0) Gecko/20100101 Firefox/100.0"`

I cannot see any other error log or stack trace.

```
=======================
 Instance Diagnostic 
=======================
APP_URL: https://p.xxx
APP_DOMAIN: p.xxx
Version: 0.11.2
PHP: 7.4.3
PHP-bcmath: Installed
PHP-ctype: Installed
PHP-curl: Installed
PHP-intl: Installed
PHP-json: Installed
PHP-mbstring: Installed
PHP-openssl: Installed
PHP-redis: Installed
Database: mysql (8.0.29-0ubuntu0.20.04.3)
Bootstrap: Writable
Storage: Writable
========= END =========
```

Any chance to get more debug info out of the system?
It runs via php 7.4 fcgi parallel to nginx.

**Comment by dansup** [comment]
How did you install this, it doesn't look like you are running the latest commits as the diagnostic data has included more data for a while now. 

Are there any logs in `storage/app/laravel.log` ?

**Comment by kkretsch** [comment]
I just did a git fetch and git checkout v0.11.2 on the exiting git clone I had. After that the usual composer install, cleaned the cache and did run the php artisan scripts like:
```
composer install --no-ansi --no-interaction --no-progress --no-scripts --optimize-autoloader
php artisan config:cache
php artisan route:cache
php artisan migrate --force
php artisan horizon:purge
php artisan storage:link
```

Anything I should have done different?
And I have no laravel.log file either in the given folder.

**Comment by dansup** [comment]
@kkretsch We released v0.11.3 recently, you should update and if you still experience these issues feel free to re-open this issue!

**Comment by mitexleo** [comment]
I'm having the same issue.. Just installed on a new server. I'm using Caddy
